### PR TITLE
Enable compile on OS X

### PR DESCRIPTION
**Pull request by chris5108**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(imu_vn_100)
 
-add_definitions("-std=c++11")
+set(CMAKE_CXX_FLAGS "-std=gnu++11")
```
Comment (kartikmohta) [inline CMakeLists.txt]: Why was this needed? Can you describe the error you get when having `std=c++11`.

File: vncpplib/include/vncp_services.h [hunk 2]
```diff
@@ -111,6 +111,17 @@ extern "C" {
...
+  #define B9600 9600
+  #define B19200 19200
+  #define B38400 38400
+  #define B57600 57600
+  #define B115200 115200
+  #define B230400 230400
+  #define B460800 460800
+  #define B921600 921600
+#endif
+
```
Comment (kartikmohta) [inline vncpplib/include/vncp_services.h]: Can you just put this block in `vncpplib/src/arch/linux/vncp_services.c` just above the `vncp_determineBaudrateFlag` function? Putting these in the header causes these macros to be defined in all the files that include this header which just pollutes the global namespace.

Comment (chris5108) [inline vncpplib/include/vncp_services.h]: Yes good point. I've pushed this change.

File: CMakeLists.txt (hunk 1 above)
Comment (chris5108) [inline CMakeLists.txt]: Running
    catkin_make --pkg imu_vn_100 --cmake-args -DCMAKE_BUILD_TYPE=Release

Results in this error on OS X:
image
